README.md

```diff
@@ -476,18 +481,50 @@ Anatomy of a Redis command
 
 All the Redis commands are defined in the following way:
 
-    void foobarCommand(client *c) {
-        printf("%s",c->argv[1]->ptr); /* Do something with the argument. */
-        addReply(c,shared.ok); /* Reply something to the client. */
+```c
+void foobarCommand(client *c) {
+    printf("%s",c->argv[1]->ptr); /* Do something with the argument. */
+    addReply(c,shared.ok); /* Reply something to the client. */
+}
+```
+
+There will be a `foobar.json` JSON file to record all the metadata for this command:
+
+```json
+{
+    "FOOBAR": {
+        "summary": "A foobar command.",
+        "complexity": "O(1)",
+        "group": "connection",
+        "since": "x.0.0",
+        "arity": 2,
+        "function": "foobarCommand",
+        "command_flags": [
+            "LOADING",
+            "FAST"
+        ],
+        "acl_categories": [
+            "CONNECTION"
+        ],
+        "arguments": [
+            {
+                "name": "message",
+                "type": "string"
+            }
+        ]
     }
+}
+```
 
-The command is then referenced inside `server.c` in the command table:
+The command is then referenced inside `commands.c`, see above file part for more details:
 
-    {"foobar",foobarCommand,2,"rtF",0,NULL,0,0,0,0,0},
+```c
+{"foobar","A foobar command.","O(1)","x.0.0",CMD_DOC_NONE,NULL,NULL,COMMAND_GROUP_CONNECTION,FOOBAR_History,FOOBAR_tips,foobarCommand,2,CMD_LOADING|CMD_FAST,ACL_CATEGORY_CONNECTION,.args=FOOBAR_Args},
+```
```

Comment:
we already have a section about commands.c (explains how it's generated).
i don't think we need to give an example of that (it's auto-generated code, not user composed).

Reply:
yes, i removed it